docs/book/src/cronjob-tutorial/testdata/project/config/prometheus/monitor_tls_patch.yaml

```diff
@@ -0,0 +1,22 @@
+# Patch for Prometheus ServiceMonitor to enable secure TLS configuration
+# using certificates managed by cert-manager
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: controller-manager-metrics-monitor
+  namespace: system
+spec:
+  endpoints:
+    - tlsConfig:
+        insecureSkipVerify: false
+        ca:
+          secret:
+            name: serving-cert
```

Comment:
TODO: We might want to create a cert for the metrics instead of re-use the webhooks one which does not seems a good practice